electricitymap/contrib/lib/tests/test_event_lists.py

```diff
@@ -47,6 +47,27 @@ def test_append_to_list_logs_error(self):
             )
             mock_error.assert_called_once()
 
+    def test_merge_exchanges(self):
+        exchange_list_1 = ExchangeList(logging.Logger("test"))
+        exchange_list_1.append(
+            zoneKey=ZoneKey("AT->DE"),
+            datetime=datetime(2023, 1, 1, tzinfo=timezone.utc),
+            netFlow=1,
+            source="trust.me",
+        )
+        exchange_list_2 = ExchangeList(logging.Logger("test"))
+        exchange_list_2.append(
+            zoneKey=ZoneKey("AT->DE"),
+            datetime=datetime(2023, 1, 1, tzinfo=timezone.utc),
+            netFlow=2,
+            source="trust.me",
+        )
+        exchanges = ExchangeList.merge_exchanges(
+            [exchange_list_1, exchange_list_2], logging.Logger("test")
+        )
+        assert len(exchanges) == 1
+        assert exchanges.events[0].netFlow == 3
```

Comment:
I think I would like one more test to check the outcome if one of these are None/Null/N/A etc. Just to make sure we have the test and avoid unexpected changes if we ever decide to refractor things in the future.

Reply:
I added a test for NaN and negative values as well.